docs/fdb_release_prep.md

```diff
@@ -0,0 +1,14 @@
+# Preparing for New Releases of FoundationDB
+
+When a new major or minor release of FoundationDB is coming up, we should take steps to ensure that the operator supports that release. This document captures some common things to look for.
+
+1. Create a label for tracking issues related to the new release.
+1. Look for new entries in the `ProcessClass.ClassType` enum in FoundationDB that are not in the `ProcessCounts` struct in the operator. File issues on making those available in the `ProcessCounts`.
+1. Look for new roles that are not in the `RoleCounts` struct in the operator. File issues on making those available in the `RoleCounts` and incorporating them into the default process counts. Some roles such as the `ratekeeper` are singletons, which means that they do not appear in `RoleCounts`, but are accounted for in the `GetProcessCountsWithDefaults` method. You can look for occurrences of `roles.addRole` in the FoundationDB code to get a quick list of the roles. Note that the role names in this search are going to be singular, but they are always plural in the `RoleCounts`.
```

Comment:
Could you link https://github.com/apple/foundationdb/blob/6.3.14/fdbrpc/Locality.h as a reference here?

Reply:
I'd rather not link to specific versions, since it creates links that we'll have to update regularly. I think the instructions will be enough to find the right reference points as the code evolves.

Reply:
We could at least mention `fdbrpc/Locality.h` and `fdbrpc/Locality.cpp` as source.